microsoft-edge/devtools-guide-chromium/css/grid.md

```diff
@@ -1,171 +1,214 @@
 
 ![Show line names](./grid-images/grid-show-line-names.png)
 
-### Checkboxes
+In the above example, 4 lines have names: `left`, `middle1`, `middle2`, and `right`.
+
+For more information about line names in the grid overlay, see [Layout using named grid lines](https://developer.mozilla.org/docs/Web/CSS/CSS_Grid_Layout/Layout_using_Named_Grid_Lines) at MDN.
+
+<!--In the demo, **orange** element spans from left to right, with `grid-column: left` and `grid-column: right` CSS.  Showing line names makes it easier to visualize the start and end position of the element.  -->
+
+
+
+<!-- ------------------------------ -->
+#### Checkboxes
 
-Select any of the checkboxes in the **Overlay display settings** section:
+The **Overlay display settings** section within the **Grid** section in the **Layout** pane contains the following checkboxes:
 
-| Option | Details |
+| Checkbox | Description |
 |:--- |:--- |
-| **Show track sizes**  | Display (or hide) the sizes of the tracks. |
-| **Show area names** | Display (or hide) the names of the area, when names are provided. |
-| **Extend grid lines** | Displays (or hides) the extensions of the grid dimensions along each axis.  By default, grid lines are only shown inside the element with `display: grid` or `display: inline-grid` CSS set on it. |
+| **Show track sizes**  | Displays or hides the sizes of the tracks. |
+| **Show area names** | Displays or hides the names of the area, when names are provided. |
+| **Extend grid lines** | Displays or hides the extensions of the grid dimensions along each axis.  By default, grid lines are only shown inside the element with `display: grid` or `display: inline-grid` CSS set on it. |
 
 The following sections explain these checkboxes.
 
-#### Show track sizes
 
-Select the **Show track sizes** checkbox, to view the track sizes of the grid.
+<!-- ---------- -->
+###### Show track sizes
+
+To view the track sizes of the grid, select the **Show track sizes** checkbox:
+
+![Show track sizes](./grid-images/grid-show-track-sizes.png)
 
-DevTools displays `[authored size]` and `[computed size]` in each line label.
+The `[authored size]` and `[computed size]` are then displayed in each line label.
 
 | Size | Details |
 |:--- |:--- |
 | **authored size** | The size defined in stylesheet (omitted if not defined). |
 | **computed size** | The actual size on screen. |
 
-In the demo, the `snack-box` column sizes are defined in the `grid-template-columns:1fr 2fr;` CSS.  Therefore, the column line labels display both authored and computed sizes.
+In the demo, the `snack-box` column sizes are defined in the `grid-template-columns:1fr 2fr;` CSS.  Therefore, the column line labels display both authored and computed sizes:
 
 | Track size | Authored size | Computed size |
 |:--- |:--- |:--- |
 | **1fr** &#x2022; **96.66px** | 1fr | 96.66px |
```

Comment:
line 132: 96.66px is probably wrong/outdated in table in section
https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/css/grid?branch=pr-en-us-3248#show-track-sizes

live:
https://learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/css/grid#show-track-sizes